src/status_im/utils/browser_config.edn

```diff
@@ -1,4 +1,4 @@
 ;; Browser (webview) configuration
 ;; :inject-js is a map of {<domain> <js-code>}, JS code will be injected if the domain name matches (www. is stripped away)
 
-{:inject-js {"cryptokitties.co" "; if (!window.chrome) { window.chrome = { webstore: true }; }"}}
+{:inject-js {"cryptokitties.co" "; if (!window.chrome) { window.chrome = { webstore: true }; window.ethereum.isMetaMask = web3.currentProvider.isMetaMask = true; }"}}
```

Comment:
Just to double check, this will only get injected in the ck web page? i.e. other sites won't mistakenly think that we are metamask?

Reply:
Never mind, just saw Julien's comment above